### PR TITLE
Feature/high resolution layout

### PR DESCRIPTION
**Pull request by BelfDev**
Tweak the Home Page layout for 1920+ resolutions.

### CODE REVIEW

File: src/pages/index.pt.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const tranlations = {
   },
   forthSection: {
     preHeadline: 'casos de',
-    headline: 'successo',
```
Comment (yagotome) [inline src/pages/index.pt.js:47]: :thinking: :rofl:

File: src/components/common/buttons/SolidButton/SolidButton.module.scss [hunk 2]
```diff
@@ -27,4 +27,19 @@
     color: $typography-on-primary-light;
   }
 
+  @include desktop {
+    padding-right: 40px;
+    padding-left: 40px;
+
+    @include media(">=#{$desktop-hr}") {
```
Comment (yagotome) [inline src/components/common/buttons/SolidButton/SolidButton.module.scss:34]: What does this `hr` stand for?

File: src/components/pages/HomePage/HomePage.module.scss [hunk 3]
```diff
@@ -175,24 +170,53 @@
...
       margin-top: 8px;
       padding-left: $content-horizontal-padding;
 
       @include desktop {
         padding: 52px 0;
 
         grid-column-gap: 4vw;
+
+        @include media(">=#{$desktop-hr}") {
+          grid-column-gap: 88px;
```
Comment (yagotome) [inline src/components/pages/HomePage/HomePage.module.scss:190]: `grid-column-gap` is not standard.

```suggestion
          column-gap: 88px;
```

File: src/components/pages/HomePage/HomePage.module.scss [hunk 4]
```diff
@@ -175,24 +170,53 @@
...
+            }
+          }
+        }
+
+        &:last-child {
+          > div:first-child > div:nth-child(2) {
+            flex-grow: unset;
+            width: 22px;
+          }
+        }
```
Comment (yagotome) [inline src/components/pages/HomePage/HomePage.module.scss:219]: This wasn't in the scope of this PR, but it's OK. We shall refactor it later in another PR.

File: src/components/common/buttons/SolidButton/SolidButton.module.scss (hunk 2 above)
Comment (BelfDev) [reply src/components/common/buttons/SolidButton/SolidButton.module.scss:34]: high-resolution

File: src/components/pages/HomePage/HomePage.module.scss (hunk 3 above)
Comment (BelfDev) [reply src/components/pages/HomePage/HomePage.module.scss:190]: Actually, we've been using `grid-column-gap` (7) more often than `column-gap` (1). Do you want to switch all occurrences to `column-gap`?

File: src/components/pages/HomePage/HomePage.module.scss (hunk 4 above)
Comment (BelfDev) [reply src/components/pages/HomePage/HomePage.module.scss:219]: Aw, come on... I thought you'd be happy with this!

File: src/components/pages/HomePage/HomePage.module.scss (hunk 3 above)
Comment (BelfDev) [reply src/components/pages/HomePage/HomePage.module.scss]: Done!

**Pull request merged**